### PR TITLE
Acceptance fileds to recruit document model

### PR DESCRIPTION
**Pull request by michalsz**

### CODE REVIEW

File: db/human_resources_migrate/20200310122458_add_acceptance_to_recruit_documents.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddAcceptanceToRecruitDocuments < ActiveRecord::Migration[6.0]
+  def change
+    add_column :recruit_documents, :accept_current_process, :boolean, default: true
+    add_column :recruit_documents, :accept_future_process, :boolean
```
Comment (schodevio) [inline db/human_resources_migrate/20200310122458_add_acceptance_to_recruit_documents.rb]: let's change name to 'accept_future_processes'

Comment (michalsz) [reply db/human_resources_migrate/20200310122458_add_acceptance_to_recruit_documents.rb]: Renamed

File: app/javascript/locales/en.js [hunk 2]
```diff
@@ -313,7 +313,9 @@ export default {
         general: 'General',
         male: 'male',
         female: 'female',
-        position: 'Position'
+        position: 'Position',
+        accept_current_process: 'Accept for recruitment',
```
Comment (schodevio) [inline app/javascript/locales/en.js]: let's use camelCase for translation keys

File: app/javascript/locales/pl.js [hunk 3]
```diff
@@ -312,7 +312,9 @@ export default {
         general: 'Ogólne',
         male: 'mężczyzna',
         female: 'kobieta',
-        position: 'Stanowisko'
+        position: 'Stanowisko',
+        accept_current_process: 'Zgodan na przetwarzanie',
```
Comment (schodevio) [inline app/javascript/locales/pl.js]: 2 typos

File: db/human_resources_migrate/20200310122458_add_acceptance_to_recruit_documents.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+class AddAcceptanceToRecruitDocuments < ActiveRecord::Migration[6.0]
+  def change
+    add_column :recruit_documents, :accept_current_process, :boolean, default: true
```
Comment (schodevio) [inline db/human_resources_migrate/20200310122458_add_acceptance_to_recruit_documents.rb]: `null: false` for both fields

File: app/javascript/locales/en.js (hunk 2 above)
Comment (michalsz) [reply app/javascript/locales/en.js]: updated

File: app/javascript/locales/pl.js (hunk 3 above)
Comment (michalsz) [reply app/javascript/locales/pl.js]: updated

File: db/human_resources_migrate/20200310122458_add_acceptance_to_recruit_documents.rb (hunk 4 above)
Comment (michalsz) [reply db/human_resources_migrate/20200310122458_add_acceptance_to_recruit_documents.rb]: added

**Pull request merged**